src/convertToObject.js

```diff
@@ -3,16 +3,39 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are
+ * the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const parts = sourceString
+    .replace(/:/g, ';')
+    .split(';')
+    .map(value => value
+      .split('')
+      .filter(symbol => symbol !== '\n')
+      .join('').trim())
+    .filter((element) => element !== '');
```

Comment:
Looks like too many unnecessary actions. Let's find a better way to do this task, without replace and nested filter, join...
Also, in the filter you don't need to compare with an empty string you can just return item.element() and all falsy values will be excluded. .filter((element) => element.trim())